sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_live_metrics.py

```diff
@@ -162,42 +169,65 @@ def __init__(self, **kwargs: Any) -> None:
             [_get_process_time_normalized],
         )
 
+
     def _record_span(self, span: ReadableSpan) -> None:
         # Only record if in post state
         if _is_post_state():
-            document = _get_span_document(span)
-            _append_quickpulse_document(document)
-            duration_ms = 0
-            if span.end_time and span.start_time:
-                duration_ms = (span.end_time - span.start_time) / 1e9  # type: ignore
-            # TODO: Spec out what "success" is
-            success = span.status.is_ok
-
-            if span.kind in (SpanKind.SERVER, SpanKind.CONSUMER):
-                if success:
-                    self._request_rate_counter.add(1)
-                else:
-                    self._request_failed_rate_counter.add(1)
-                self._request_duration.record(duration_ms)
-            else:
-                if success:
-                    self._dependency_rate_counter.add(1)
+            try:
+                document = _get_span_document(span)
+                _append_quickpulse_document(document)
+                duration_ms = 0
+                if span.end_time and span.start_time:
+                    duration_ms = (span.end_time - span.start_time) / 1e9  # type: ignore
+                # TODO: Spec out what "success" is
```

Comment:
Agree on this comment - I seem to have been operating on an older definition of what success means, where client spans success is defined differently than server span. A few months ago this definition was modified slightly: https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/commit/4136596bbc8f1663970cf14393a8e7e87481b7d6. Let me know what you think about success definitions & once that's solid I can go back and change Node.js live metrics code if needed to reflect that.

Reply:
No need to decide on the definition here. I think it is fine to iterate on the specs you linked and then retroactively modify live metrics implementation after. For now (at least for this PR), we use the definition of success that the breeze exporters use as well.